src/cl/prysm/prysm_launcher.star

```diff
@@ -339,14 +339,14 @@ def get_validator_config(
 		# we do the for loop as otherwise its a proto repeated array
 		cmd.extend([param for param in extra_params])
 
-
+	NODE_ARTIFACT_UUID = node_keystore_files.files_artifact_uuid if node_keystore_files != None else package_io.NO_ARTIFACT_UUID
```

Comment:
similar comment as above for lighthouse, i am not sure if this works with NO_ARTIFACT_UUID, lets do the if condition